utoipa-swagger-ui/src/oauth.rs

```diff
@@ -0,0 +1,81 @@
+use std::collections::HashMap;
```

Comment:
This could also have some module level comments like
```rust
//! Implements Swagger UI [oauth configuration](https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/oauth2.md) options.
```